packages/ecs-init/README.md

```diff
@@ -0,0 +1,24 @@
+# ecs-init
+
+The ecs-init package in Bottlerocket provides a program and systemd unit for
+launching and supervising the ECS agent.
+
+This README is temporary and is meant to track the known issues and remaining
+work items for ecs-init.
+
+## Known issues
+
+* Cache broken - The package includes a cached copy of the ECS agent at
+  `/var/cache/ecs`.  However, Bottlerocket's filesystem does not have this
+  visible.
+* Nvidia library patched out - Runtime errors occur due to a missing symbol,
+  even when compiled with the recommended linker options:
+  `/x86_64-bottlerocket-linux-gnu/sys-root/usr/libexec/amazon-ecs-init: undefined symbol: nvmlDeviceGetGraphicsRunningProcesses`.
+  For now, a patch removes the offending code.
+* Plugin directories skipped - ecs-init attempts to bind-mount plugin-related
+  directories into the agent container.  Docker attempts to automatically
+  create any directory that does not exist, and `/usr/lib/docker/plugins` is
+  such a directory that does not exist.  However, Bottlerocket's filesystem is
+  mostly read-only, and ecs-init fails to create this directory.  For now,
+  bind-mounting is skipped when a directory does not exist, but this will break
```

Comment:
I'm not familiar with how plugins are used, what functionality would this limit?

Reply:
Docker plugins can be installed into those directories, see [here](https://docs.docker.com/engine/extend/plugin_api/#plugin-discovery).  The agent scans these directories to discover unmanaged plugins.  However, it does look like "managed" plugins (installed via the "docker plugin install" command) are typically installed into `/run/docker/plugins` which does work on Bottlerocket.  It's probably reasonable for us to only support managed plugins on Bottlerocket, or unmanaged plugins built into Bottlerocket itself.